### PR TITLE
Basic i18n formatting page

### PR DESCRIPTION
**Pull request by rtymchyk**

### CODE REVIEW

File: 13-i18n.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# i18n
+
+## Formatting 
+It's a good idea to read [this](https://crowdtilt.atlassian.net/wiki/pages/viewpage.action?pageId=60850186) to understand how locale generally works on desktop/web
```
Comment (tildedave) [inline 13-i18n.md]: This repo is public so please summarize this in this document

**Comment by tildedave** [comment]
I see a lot of "don't use X" here - maybe better to walk through what the challenges are (e.g. EVERY DATE PRICE AND NUMBER NEEDS TO PASS THROUGH A FORMATTER) and then how we solve it.

**Comment by rtymchyk** [comment]
@tildedave Completely forgot this is public! I will clean it up.

**Comment by rtymchyk** [comment]
@amackera @patrick-tilt

**Comment by rtymchyk** [comment]
ping Merging this if nobody has anything else to say

**Pull request merged**